content/docs/concepts/multicluster-deployments/index.md

```diff
@@ -0,0 +1,119 @@
+definitions of the same service which will need to be reconciled when integrating the clusters.
+
+A multicluster mesh has the advantage that all the services look the same to clients,
+regardless of where the workloads are actually running. It's transparent
+to the application whether it's deployed in a single or multicluster mesh.
+To achieve this behavior, a single logical control plane needs to manage all services,
+however, the single logical control plane doesn't necessarily need to be a single physical
+Istio control plane. There are two possible deployment approaches:
+
+1. A single Istio control plane that can access and configure all the services in the mesh.
+
+1. Multiple synchronized Istio control planes that have replicated service and routing configurations.
+
+Even within these two topologies, there is more than one way to configure a multicluster mesh.
+Which approach to use and how to configure it depends on the requirements of the application
+and on the features and limitations of the underlying cloud deployment platform.
+
+### Single control plane topology
+
+This multicluster configuration uses a single Istio control plane running on one of the clusters.
+The control plane's Pilot manages services on the local and remote clusters and configures the
+Envoy sidecars for all of the clusters. This approach works best in environments where all of
+the participating clusters have VPN connectivity so every pod in the mesh is reachable from anywhere
+else using the same IP address.
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-with-vpn.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters with direct network access to remote pods over VPN"
+    >}}
+
+In this configuration, the Istio control plane is deployed on one of the clusters while all other
+clusters run a simpler remote Istio configuration which connects them to the single Istio control plane
+that manages all of the Envoy's as a single mesh. The IP addresses on the various clusters must not
+overlap and note that DNS resolution for services on remote clusters is not automatic.
+Users need to replicate the services on every participating cluster.
+You can find detailed steps to set up this kind of multicluster topology
+in our [single control plane with VPN instructions](/docs/setup/kubernetes/multicluster-install/vpn/).
+
+If setting up an environment with universal pod-to-pod connectivity is difficult or impossible,
+it may still be possible to configure a single control plane topology using Istio gateways and
+configuring Istio Pilot to enable location-aware service routing (a.k.a. split-horizon EDS).
+
+{{< image width="80%" ratio="36.01%"
+    link="./multicluster-split-horizon-eds.svg"
+    caption="Istio mesh spanning multiple Kubernetes clusters using single Istio control plane and Gateway to reach remote pods"
+    >}}
+
+In this configuration, the control plane forwards a request from a sidecar in one cluster to a service in
```

Comment:
it almost sounds as if the control plane is involved in request routing. I suggest massaging this piece and the para